snapfaas/src/syscalls.proto

```diff
@@ -162,6 +159,7 @@ message Syscall {
     BlobOpen openBlob = 16;
     BlobRead readBlob = 17;
     BlobClose closeBlob = 18;
-    ReadDir readDir = 19;
+    Component declassify = 19;
+    ReadDir readDir = 20;
```

Comment:
`readDir` should remain `19`, it's bad to change protobuf field/variant numbering unnecessarily